docs/howto/smart_contract_upload.md

```diff
@@ -0,0 +1,138 @@
+# Smart Contract Upload
+
+This tutorial covers how to create a circuit on a splinter network between 
+two nodes and upload a smart contract via the circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running. The tutorial will refer
+  to the two nodes as "alpha" and "beta" respectively
+* The scabbard command line tool installed
+* The splinter command line tool installed
```

Comment:
Should this mention that experimental features need to be enabled?

Reply:
Updated, to specify that cli tools need to be experimental features.